src/main/java/uk/gov/laa/ccms/caab/bean/ClientFormDataAddressDetails.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.laa.ccms.caab.bean;
+
+import lombok.Data;
+
+/**
+ * Represents the client address details form.
+ */
+@Data
+public class ClientFormDataAddressDetails {
+
+  private Boolean noFixedAbode = false;
+  private String country;
+  private String houseNameNumber;
+  private String postcode;
+  private String addressLine1;
+  private String addressLine2;
+  private String cityTown;
+  private String county;
+
+  //Required for address searching
+  private ClientFormDataAddressSearch addressSearch;
+
+  //Required for validation
+  private Boolean vulnerableClient = false;
+
+  //Required for template rendering
+  private String clientFlowFormAction;
```

Comment:
Was there no way this (and possibly vunerableClient) could be pushed into an abstract superclass?

Reply:
amended